torch/csrc/Module.cpp

```diff
@@ -269,6 +269,17 @@ PyObject *THPModule_addDocStr(PyObject *_unused, PyObject *args)
           "Type '%s' already has a docstring", t->tp_name);
     }
     t->tp_doc = doc_str;
+  } else if (strcmp(Py_TYPE(obj)->tp_name, "OpOverloadPacket") == 0) {
+    const auto current_doc_str = PyObject_GetAttrString(obj, "__doc__");
+    if (current_doc_str != Py_None) {
+      const auto qual_op_name_obj = PyObject_GetAttrString(
+        obj, "qualified_op_name");
+      const auto qual_op_name_str = THPUtils_unpackString(
+        qual_op_name_obj);
+      return PyErr_Format(PyExc_RuntimeError,
+          "function '%s' already has a docstring", qual_op_name_str.c_str());
+    }
+    PyModule_SetDocString(obj, doc_str);
```

Comment:
Is this actually OK, I'm pretty sure obj is not a module :o

Reply:
looking at cpython it seems to be correct

```
int
PyModule_SetDocString(PyObject *m, const char *doc)
{
    PyObject *v;

    v = PyUnicode_FromString(doc);
    if (v == NULL || _PyObject_SetAttrId(m, &PyId___doc__, v) != 0) {
        Py_XDECREF(v);
        return -1;
    }
    Py_DECREF(v);
    return 0;
}
```

Reply:
yeah, not sure why this is named after `Module`.  Ito does seem to work properly and the source is not targeting only modules.